build.gradle

```diff
@@ -1,6 +1,7 @@
 apply from: './dependencies.gradle'
 
 buildscript {
+    ext.kotlin_version = '1.3.61'
```

Comment:
Instead of doing it here you should change the version in the file dependencies.gradle.